docs/src/explanations/memories.md

```diff
@@ -24,8 +24,12 @@ def sin(t: Double): Double = t // What should this be?
 For example, users can create a small ROM initialized to 1, 2, 4, 8 and loop through all values using a counter as an address generator as follows:
 
 ```scala mdoc:compile-only
-val m = VecInit(1.U, 2.U, 4.U, 8.U)
-val r = m(counter(m.length.U))
```

Comment:
I finally see what's going on here... Can you also remove `scala mdoc:invisible` block above. That was what was defining `counter`, but just not showing it. Not ideal.

Reply:
Done